Dockerfile

```diff
@@ -100,6 +100,9 @@ COPY --from=builder /usr/local/bin/wasm-opt /usr/local/bin
 #
 FROM base-optimizer as rust-optimizer
 
+# Download the crates.io index using the new sparse protocol to improve performance
+ENV CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
```

Comment:
Looks good. But when you add it here it only works for `rust-optimizer`, not `workspace-optimizer`. See

```
$ docker inspect cosmwasm/rust-optimizer-arm64:sparse-testing | jq '.[0].Config.Env'
[
  "PATH=/usr/local/cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
  "RUSTUP_HOME=/usr/local/rustup",
  "CARGO_HOME=/usr/local/cargo",
  "RUST_VERSION=1.68.0",
  "CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse",
  "RUSTC_WRAPPER=sccache"
]

$ docker inspect cosmwasm/workspace-optimizer-arm64:sparse-testing | jq '.[0].Config.Env'
[
  "PATH=/usr/local/cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
  "RUSTUP_HOME=/usr/local/rustup",
  "CARGO_HOME=/usr/local/cargo",
  "RUST_VERSION=1.68.0"
]
```

When you move it one block up to base-optimizer, it is inherited by both images automatically.

Reply:
Thanks, I didn't notice I had placed it in the wrong position. Fixed.